lib/galaxy/model/store/ro_crate_utils.py

```diff
@@ -220,7 +220,77 @@ def _add_workflows(self, crate: ROCrate):
 
             crate.license = self.workflow.license or ""
             crate.mainEntity["name"] = self.workflow.name
-            crate.mainEntity["subjectOf"] = cwl_wf
+
+            # Adding the creator information
+            if hasattr(self.workflow, 'creator_metadata') and self.workflow.creator_metadata:
+                creators = self.workflow.creator_metadata
+                if creators and isinstance(creators, list) and len(creators) > 0:
```

Comment:
You've already checked this above. If creators was an empty list you wouldn't get here.